src/els_folding_range_provider.erl

```diff
@@ -42,10 +38,10 @@ handle_request({document_foldingrange, Params}, State) ->
 %% Internal functions
 %%==============================================================================
 
--spec folding_range(poi()) -> folding_range().
-folding_range(#{range := #{from := {FromL, FromC}, to := {ToL, ToC}}}) ->
-  #{ startLine      => FromL
-   , startCharacter => FromC
-   , endLine        => ToL
-   , endCharacter   => ToC
+-spec folding_range(poi_range()) -> folding_range().
+folding_range(#{from := {FromLine, FromCol}, to := {ToLine, ToCol}}) ->
```

Comment:
This does not seem to work for me:

<img width="826" alt="Screenshot 2020-03-08 at 15 47 45" src="https://user-images.githubusercontent.com/91769/76165075-65208a80-6154-11ea-8de3-8ea8658af9ab.png">

Reply:
Have you cleared the cache? Otherwise the POIs don't get updated unless you change the file.

Reply:
@jfacorro Yes, we should bump the DB version for this.

Reply:
Tried after clearing the cache, it seems to work correctly in Emacs, even if the `...` is appended to the first line instead of being on a new line, but I'm sufficiently happy with that. :)